res/database/Default/user/scr0llbaer_screenplays_adaptations.xml

```diff
@@ -533,7 +548,6 @@
 			</title>
 			<title_original>
 				<all>Baudolino</all>
-				<it>Baudolino</it>
```

Comment:
because for "it" it would use the fallback then ("en"). ?

Reply:
Yes. It makes no real difference, but the explicit language value indicates that the title is correct.

Reply:
Good point. Yes I guess I still felt a bit unsure what is "correct" in these cases.

Reply:
Ok I feel now best practice should be that `_original` tags generally should *not* have `<all>`. They are, after all, for documentary purposes, and specifying an entry for each language, even if it is the same, means that it has been looked up and confirmed that the original title in a particular language is indeed xyz. Part of that information is lost when `<all>` is used. For the according spoof titles then, it may be more a matter of taste if the languages in the spoof titles should be kinda mapped 1-on-1 or `<all>` is preferred.